test_discovery.py

```diff
@@ -160,39 +160,57 @@ def test_single_entity(entities, test_name, test_value):
     return (0, 0)
 
 
-def test_single_case(test):
+def test_single_case(test_dict, response_intent_dict):      #most_likely_intent
```

Comment:
Could you move this comment to the comment section below and add more explanation?